.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "third_party/protozero"]
 	path = third_party/protozero
 	url = https://github.com/mapbox/protozero.git
+[submodule "third_party/microtar"]
+	path = third_party/microtar
+	url = https://github.com/rxi/microtar.git
```

Comment:
Is this module well tested? I didn't see any tests in this repo. Have we used it elsewhere or have experience with it?

Reply:
I've used it before, over in https://github.com/Project-OSRM/osrm-backend

The main use-case here is writing small `.tar` files for tests.  Valhalla has its own `.tar` reading code, and does not write `.tar` files itself.

Reply:
Okay. My concern was if this module was well-tested (it doesn't have any tests in the repo) and if not, has it had any soak time in any of our projects.